### ISSUE TITLE
Add CONTRIBUTING

### ISSUE THREAD

**Issue by plumenator** [tssm/up-to-date-real-world-haskell#22]
Sorry, I didn't know how else to contact you. It would be great if there was a way to contribute. I for one, am interested. :-)

**Comment by supermacro** [comment]
In particular, I'm not familiar with the `.org` file extension. Is there anything I should be aware of when wanting to submit patches to this kind of file?

**Comment by aodhneine** [comment]
Agreed, it would be nice to have some sort of contributing guidelines for people who are interested and want to help.

Other than that, I would advise to first read Org manual (see https://orgmode.org/manual/index.html), and also look through existing code to see how it's formatted and structured.

Also, you can start by reading through original chapters of the book, trying the examples in them and fixing them if they don't work. You can just fork the repository and submit pull request with fixed example.

**Comment by tssm** [comment]
Thank you for your interest in the project, folks! I wrote my thoughts on how to continue from now on, they are available in #31. English is not my first language so if you have time please review it. Keep in mind that I'm open to adopt whatever guideline that makes contributing easier for everyone so let me know what you think.
